modules/ting/ting.install

```diff
@@ -85,6 +85,8 @@ function ting_schema() {
     ),
   );
 
+  // BBS-SAL: move/copy this to opensearch. Rename table to cache_opensearch
```

Comment:
Please note that DDB has another ticket on refactoring the cache which i planed to after SAL has been added to core. So this will be fixed later on :-)

Reply:
Place note this is true for all the cache relate stuff in the ting module.

Reply:
@cableman: Are you saying that we should or should not move the cache as a part of this PR?

Reply:
Yes, I think this should wait as much as this cache may very well be rewritten i the near feature.

If I'am ending up with the task of re-doing caching for the data-well. I would start for the start an remove all the old legacy caching and re-think it.

Reply:
:+1: